### PR TITLE
fix(sigver): remove antipatterns

### PR DESCRIPTION
**Pull request by nksazonov**

### CODE REVIEW

File: pkg/universal_sigver/client.go [hunk 1]
```diff
@@ -47,24 +47,24 @@ type Client interface {
...
 
-func NewUniversalSigVer(providerURL string, smartWalletConfig *smart_wallet.Config, entryPointAddress *common.Address) Client {
+func NewUniversalSigVer(providerURL string, smartWalletConfig *smart_wallet.Config, entryPointAddress common.Address) (Client, error) {
 	provider, err := ethclient.Dial(providerURL)
 	if err != nil {
-		panic(fmt.Errorf("failed to connect to Ethereum node: %w", err))
+		return nil, fmt.Errorf("failed to connect to Ethereum node: %w", err)
 	}
 
 	var entryPointAddress_ = entryPointAddress
```
Comment (MaxMoskalenko) [inline pkg/universal_sigver/client.go:59]:
```suggestion
	if entryPointAddress.Cmp(common.Address{}) == 0 {
		entryPointAddress = entryPointV0_6Address
	}
```

Comment (MaxMoskalenko) [reply pkg/universal_sigver/client.go:59]: We don't need entryPointAddress_ if we are not using pointers

Comment (nksazonov) [reply pkg/universal_sigver/client.go]: [20115a3](https://github.com/layer-3/clearsync/pull/268/commits/20115a32ebbaeea813cef609d9bb3eb194b82a65)

**Comment by nksazonov** [comment]
Moved to #270.

**Pull request closed without merging**